blog/2021-12-14-cohort-based-modelling/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Cohort-based modelling with Apache COOL
```

Comment:
Do not use the Apache COOL now, because the COOL has not been admitted to Apache till now.

Reply:
add more content about the AI part.